lisa/utils.py

```diff
@@ -440,37 +433,41 @@ def from_map(cls, mapping):
     def from_yaml_map(cls, path):
         """
         Allow reloading from a plain mapping, to avoid having to specify a tag
-        in the configuration file. The content is hosted under a top-level key
-        specified in :attr:`YAML_MAP_TOP_LEVEL_KEY`.
+        in the configuration file. The content is hosted under the top-level
+        key specified in :attr:`STRUCTURE`.
```

Comment:
That should be
```suggestion
        key specified in ``STRUCTURE``.
```
(the ref can't be resolved)

Reply:
ok, if we have no better way to refer to class attributes let's use the same convention as instance ones